client/my-sites/media-library/scale.jsx

```diff
@@ -123,12 +123,18 @@ class MediaLibraryScale extends Component {
 		return (
 			<div className="media-library__scale">
 				<SegmentedControl className="media-library__scale-toggle" compact>
-					<SegmentedControlItem selected={ 1 !== scale } onClick={ this.setScaleToMobileGrid }>
-						<span className="media-library__scale-toggle-label">{ translate( 'Grid' ) }</span>
+					<SegmentedControlItem
```

Comment:
Since this toolbar simply changes visual representation, it's not really useful for screen reader users. How about hiding it completely with `aria-hidden="true"` ?